### PR TITLE
TECH-1657: Replace jahia-discovery with jahia-ee

### PR DESCRIPTION
**Pull request by Fgerthoffert**
- Updated docker image
- Updated the list of modules to be installed (using fixed version)
- Cleaned incorrect workflow parameters
- Minor tweak in the test
- Fixed an incorrect configuration for fetching the list of module (was in error before)

### CODE REVIEW

File: tests/provisioning-manifest-build.yml [hunk 1]
```diff
@@ -1,2 +1,22 @@
-# nothing to add
-- sleep: 10
+- addMavenRepository: 'https://store.jahia.com/nexus/content/repositories/jahia-public-app-store@id=JahiaStore'
+- addMavenRepository: 'https://devtools.jahia.com/nexus/content/groups/public/@snapshots@noreleases@id=JahiaSnapshot'
+
+# This modules were added during the switch from using jahia-discovery to jahia-ee
+- installBundle: 
+  - 'mvn:org.jahia.modules/press/3.1.0'
```
Comment (rknj) [inline tests/provisioning-manifest-build.yml:6]: I think we're missing the digitall module here

File: tests/provisioning-manifest-snapshot.yml [hunk 2]
```diff
@@ -1,7 +1,30 @@
+- addMavenRepository: 'https://store.jahia.com/nexus/content/repositories/jahia-public-app-store@id=JahiaStore'
+- addMavenRepository: 'https://devtools.jahia.com/nexus/content/groups/public/@snapshots@noreleases@id=JahiaSnapshot'
+
 - addMavenRepository: 'https://devtools.jahia.com/nexus/content/groups/internal@id=jahia-internal@snapshots'
   username: ${env:NEXUS_USERNAME}
   password: ${env:NEXUS_PASSWORD}
 
+# This modules were added during the switch from using jahia-discovery to jahia-ee
+- installBundle: 
+  - 'mvn:org.jahia.modules/press/3.1.0'
```
Comment (rknj) [inline tests/provisioning-manifest-snapshot.yml:10]: we're missing the digitall module here too

**Review by rknj — CHANGES_REQUESTED** [review]

File: tests/provisioning-manifest-build.yml (hunk 1 above)
Comment (Fgerthoffert) [reply tests/provisioning-manifest-build.yml:6]: The tests are passing without it, so I'd be in favor of not adding it except if required by the tests.

File: tests/provisioning-manifest-snapshot.yml (hunk 2 above)
Comment (Fgerthoffert) [reply tests/provisioning-manifest-snapshot.yml:10]: The tests are passing without it, so I'd be in favor of not adding it except if required by the tests.

**Review by rknj — APPROVED** [review]

**Pull request merged**